src/routes/token/mintToken.ts

```diff
@@ -0,0 +1,84 @@
+import { RequestHandler } from 'express';
+import SorobanClient from 'soroban-client';
+import dotenv from 'dotenv';
+
+import address from '../../models/address';
+import {
+  transactionMintBuild,
+  transactionBalanceBuild,
+} from '../../utils/token/transactionBuild';
+import getToken from '../../utils/token/getToken';
+import returnObj from '../../utils/returnObj';
+import getTransaction from '../../utils/token/getTransaction';
+import addAddressToDb from '../../utils/address/addAddressToDb';
+import { responseTransaction } from '../../utils/interfaces';
+
+dotenv.config();
+
+const server = new SorobanClient.Server(process.env.SOROBAN_SERVER);
+
+const fee = 100000;
+
+const mintToken: RequestHandler = async (req, res) => {
+  try {
+    const { addressUser } = req.body;
+
+    const findAddress = await address.findOne({ address: addressUser });
+    if (findAddress) {
+      return res.json(returnObj(false, 'address exist in database', null));
+    }
+
+    const adminAddress: string | undefined = process.env.ADMIN_ADDRESS;
```

Comment:
remove the env
use getFee
use getServer
change addressUser to `user`
rename findAddress to isUserAlreadyMinted
change the error message to 'User has already minted tokens'

use getAdmin